README.md

```diff
@@ -34,27 +34,29 @@ informative examples on provided clusters.
     - Distributed deployment
     - Cluster computing exercises
 
-## Chat Room
-
-Stuck? Ask for help here: https://gitter.im/dask/pydata-dc-2016 
 
 ## Installation
 
 1. Install [Anaconda](https://www.continuum.io/downloads)
-2. Update select packages
+2. Create a new conda environment:
+
+        conda create -n parallel jupyter scikit-learn pandas pytables matplotlib ipyparallel joblib dask distributed bokeh ujson snakeviz
```

Comment:
To be honest I've never used environment files.  I'm happy to defer to others' judgement here.